src/helpers/txHelper.ts

```diff
@@ -15,6 +15,66 @@ import { LedgerErrorResponse } from "@/types/types";
 
 import { validate } from "@/validate";
 
+/* Build Transaction related */
+export type FeeBumpedTxResponse = {
+  errors: string[];
+  xdr: string;
+};
+
+const buildFeeBumpTx = ({
+  innerTxXDR,
+  maxFee,
+  sourceAccount,
+  networkPassphrase,
+}: {
+  innerTxXDR: string;
+  maxFee: string;
+  sourceAccount: string;
+  networkPassphrase: string;
+}): FeeBumpedTxResponse => {
+  const result = {
+    errors: [] as string[],
+    xdr: "",
+  };
+
+  let innerTx: Transaction;
+
+  try {
+    innerTx = TransactionBuilder.fromXDR(
+      innerTxXDR,
+      networkPassphrase,
+    ) as Transaction;
+  } catch (e) {
+    result.errors.push("Invalid inner transaction XDR.");
+    return result;
+  }
+
+  if (typeof innerTx?.operations === "undefined") {
+    result.errors.push("Inner transaction must be a regular transaction.");
+    return result;
+  }
+
+  let feeBumpTx;
+
+  try {
+    feeBumpTx = TransactionBuilder.buildFeeBumpTransaction(
```

Comment:
If we're not using `feeBumpTx` outside of this `try` block, we can make this `const`.